app/src/main/res/values-fr/strings.xml

```diff
@@ -9,8 +9,8 @@
         <item name="valor">"Bravoure"</item>
         <item name="instinct">"Intuition"</item>
     </string-array>
-    <string name="place1">"Dans l\'ensemble, ton Pokémon&#8230;"</string>
-    <string name="place2">"Ses stats&#8230;"</string>
+    <string name="place1">Dans l\'ensemble, ton Pokémon&#8230;</string>
+    <string name="place2">Ses stats&#8230;</string>
```

Comment:
`<string name="place1">"Dans l\'ensemble, ton Pokémon&#8230;"</string>`
This code does not show a `\'` on my phone. Which is weird, because I think I remember having to delete the `\` from the strings that are between quotation marks for this exact reason.